.travis.yml

```diff
@@ -13,26 +16,27 @@ services:
     - docker
 
 matrix:
-   include:
-        - python: 3.6
-          env: PYTHONASYNCIODEBUG=
-          addons:
-              mariadb: 5.5
-        - python: 3.6
-          env: PYTHONASYNCIODEBUG=1
-          addons:
-              mariadb: 10.0
-        - python: 3.6
-          env: PYTHONASYNCIODEBUG=
-          addons:
-              mariadb: 10.1
-        - python: 3.6
-          env: PYTHONASYNCIODEBUG=
-          addons:
-              mysql: 5.7
+    include:
```

Comment:
Please revert the indentation changes. They are not related and make the review harder.